pages/home.md

```diff
@@ -8,88 +8,162 @@ permalink: /
   <div class="hero-body">
     <div class="container">
       <h1 class="title has-text-centered is-size-3 is-size-2-tablet is-size-1-desktop">
-        The Web — <a href="{{site.baseUrl}}/take3" title="Why Take 3?">Take 3</a>
+        Solid
```

Comment:
Here, the idea was that `Solid` is already present in the top left hand corner of the page, so the space could be used for a tagline, rather than repeating `Solid`. Moreover, this linked to the description of the intent of Solid by Tim, which sends a powerful message.

Reply:
We're going to add a vision statement back in under the roadmap section, in conjunction with some history copy that tells the story well. We could look at swapping the name with the logo here instead.

Reply:
@NSeydoux replaced the "Solid" text with the logo. Looks nicer this way ✅